### PR TITLE
fix: a11y issues fixes

### PR DESCRIPTION
**Pull request by Kyzyl-ool**

### CODE REVIEW

File: src/components/DefinitionList/DefinitionList.tsx [hunk 1]
```diff
@@ -109,6 +121,9 @@ export function DefinitionList({
                                 text={copyText}
                                 className={b('copy-button')}
                                 view={iconInside ? 'raised' : 'flat-secondary'}
+                                extraProps={{
+                                    'aria-label': i18n('label_copy'),
```
Comment (amje) [inline src/components/DefinitionList/DefinitionList.tsx:125]: ClipboardButton already has aria text

File: src/components/SharePopover/SharePopover.tsx [hunk 2]
```diff
@@ -103,6 +105,7 @@ export const SharePopover = (props: SharePopoverProps) => {
         renderCopy,
         children,
         onClick,
+        controlAriaLabel,
```
Comment (amje) [inline src/components/SharePopover/SharePopover.tsx:108]: `buttonAriaLabel`, we already have button* props

**Review by amje — CHANGES_REQUESTED** [review]

File: src/components/DefinitionList/DefinitionList.tsx (hunk 1 above)
Comment (Kyzyl-ool) [reply src/components/DefinitionList/DefinitionList.tsx]: really, thank you

File: src/components/SharePopover/SharePopover.tsx (hunk 2 above)
Comment (Kyzyl-ool) [reply src/components/SharePopover/SharePopover.tsx]: ok

**Review by amje — APPROVED** [review]

**Review by ogonkov — APPROVED** [review]

**Pull request merged**